From f6a82a27a0481b761218a5208134c7a72ec8206a Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fabian@ritter-vogt.de>
Date: Sat, 3 Mar 2018 20:07:16 +0100
Subject: [PATCH] Don't quit on SIGHUP

SIGHUP is sent to daemons on reload, so they should reread their configuration.
Currently sddm would just quit on reload, which is very unexpected.
---
 src/daemon/DaemonApp.cpp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/daemon/DaemonApp.cpp b/src/daemon/DaemonApp.cpp
index b5c8d49..de78e75 100644
--- a/src/daemon/DaemonApp.cpp
+++ b/src/daemon/DaemonApp.cpp
@@ -68,8 +68,7 @@ namespace SDDM {
         // initialize signal signalHandler
         SignalHandler::initialize();
 
-        // quit when SIGHUP, SIGINT, SIGTERM received
-        connect(m_signalHandler, SIGNAL(sighupReceived()), this, SLOT(quit()));
+        // quit when SIGINT, SIGTERM received
         connect(m_signalHandler, SIGNAL(sigintReceived()), this, SLOT(quit()));
         connect(m_signalHandler, SIGNAL(sigtermReceived()), this, SLOT(quit()));
 
--
2.16.1
